pkg/microservice/aslan/core/service/handler/router.go

```diff
@@ -77,6 +77,7 @@ func (*Router) Inject(router *gin.RouterGroup) {
 
 	pm := router.Group("pm")
 	{
+		pm.PUT("/healthCheckUpdate", gin2.UpdateOperationLogStatus, UpdateServiceHealthCheckStatus)
```

Comment:
Need it to be authorized to use this API?

Reply:
yes , there was a mismatch